From dcec4ca91eaba6301b00ffa78df5f7847eb7a81d Mon Sep 17 00:00:00 2001
From: Matthew Sakai <msakai@redhat.com>
Date: Thu, 27 Oct 2022 22:11:44 -0400
Subject: [PATCH 20/21] Replace prandom_bytes() with get_random_bytes().

.../random.h
Replaced uses of prandom_bytes() with get_random_bytes(). As
  of kernel version 5.19, prandom_bytes() no longer has a separate
  implementation but instead calls get_random_bytes() directly.
  get_random_bytes() has existed since at least kernel version 5.0,
  however, so there should not be any backwards-compatibility issues
  with using it.
---
 uds/random.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/uds/random.h b/uds/random.h
index eae36ccb..cf512523 100644
--- a/uds/random.h
+++ b/uds/random.h
@@ -56,7 +56,7 @@ void randomCompileTimeAssertions(void);
 #ifdef __KERNEL__
 static INLINE void fillRandomly(void *ptr, size_t len)
 {
-  prandom_bytes(ptr, len);
+  get_random_bytes(ptr, len);
 }
 #else
 void fillRandomly(void *ptr, size_t len);
-- 
2.38.1

